<!DOCTYPE HTML>
<!--
	Solid State by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Computational Biomedicine Group | Finotello lab | Bioinformatics | Computational Biology | Cancer immunology | University of Innsbruck</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">

		<!-- Page Wrapper -->
			<div id="page-wrapper">

				<!-- Header -->
					<header id="header">
						<h1><a href="index.html">Computational Biomedicine Group</a></h1>
						<nav>
							<a href="#menu">Menu</a>
						</nav>
					</header>

				<!-- Menu -->
					<nav id="menu">
						<div class="inner">
							<h2>Menu</h2>
							<ul class="links">
								<li><a href="index.html">Home</a></li>
								<li><a href="team.html">Team</a></li>
								<li><a href="projects.html">Projects</a></li>
								<li><a href="publications.html">Publications</a></li>
								<li><a href="software.html">Software</a></li>
								<li><a href="teaching.html">Teaching</a></li>
								<li><a href="events.html">Events</a></li>
								<li><a href="funding.html">Funding</a></li>	
							</ul>
							<a href="#" class="close">Close</a>
						</div>
					</nav>

				<!-- Wrapper -->
					<section id="wrapper">
						<header>
							<div class="inner">
								<h2>Publications</h2>
								<p>Computational Biomedicine Group</p>
							</div>
						</header>

						<!-- One -->
						<section id="one" class="wrapper alt style1">
							<div class="inner">
								<h2 class="major">Selected Publications</h2>
								<section class="features">
									<article>
										<a href="https://doi.org/10.1016/j.isci.2025.113448" class="image"><img src="images/NovumRNA_ga.jpg" alt="" /></a>
										<h3 class="major">NovumRNA: Accurate prediction of non-canonical tumor antigens from RNA sequencing data</h3>
										<p>Ausserhofer M. et al. <b>iScience</b> (2025).</p>
										<a href="https://doi.org/10.1016/j.isci.2025.113448" class="special">Read article</a>
									</article>
									<article>
										<a href="https://doi.org/10.1016/j.isci.2024.110529" class="image"><img src="images/iscience2024.jpg" alt="" /></a>
										<h3 class="major">Multimodal analysis unveils tumor microenvironment heterogeneity linked to immune activity and evasion</h3>
										<p>Lapuente-Santana O., Sturm G. et al. <b>iScience</b> 27(8), 110529 (2024).</p>
										<a href="https://doi.org/10.1016/j.isci.2024.110529" class="special">Read article</a>
									</article>
									<article>
										<a href="https://doi.org/10.1016/j.patter.2021.100293" class="image"><img src="images/patterns2021.png" alt="" /></a>
										<h3 class="major">Interpretable systems biomarkers predict response to immune-checkpoint inhibitors</h3>
										<p>Lapuente-Santana O. et al. <b>Patterns</b> 2(8), 100293 (2021).</p>
										<a href="https://doi.org/10.1016/j.patter.2021.100293" class="special">Read article</a>
									</article>
									<article>
										<a href="https://doi.org/10.1038/s41576-019-0166-7" class="image"><img src="images/natrevgen2020.png" alt="" /></a>
										<h3 class="major">Next-generation computational tools for interrogating cancer immunity. </h3>
										<p>Finotello F. et al. <b>Nature Reviews Genetics</b> 20(12), 724–746 (2019).</p>
										<a href="https://doi.org/10.1038/s41576-019-0166-7" class="special">Read article</a>
									</article>
									<article>
										<a href="https://doi.org/10.1186/s13073-019-0638-6" class="image"><img src="images/genmed2019.png" alt="" /></a>
										<h3 class="major">Molecular and pharmacological modulators of the tumor immune contexture revealed by deconvolution of RNA-seq data</h3>
										<p>Finotello F. et al. <b>Genome Medicine</b> 11(1), 34 (2019).</p>
										<a href="https://doi.org/10.1186/s13073-019-0638-6" class="special">Read article</a>
									</article>
									<article>
										<a href="https://doi.org/10.1016/j.celrep.2016.12.019" class="image"><img src="images/tciaweb.png" alt="" /></a>
										<h3 class="major">Pan-cancer Immunogenomic Analyses Reveal Genotype-Immunophenotype Relationships and Predictors of Response to Checkpoint Blockade</h3>
										<p>Charoentong P.*, Finotello F*,  Angelova M.* et al. <b>Cell Reports</b> 18(1), 248-262 (2017).</p>
										<a href="https://doi.org/10.1016/j.celrep.2016.12.019" class="special">Read article</a>
									</article>
								</section>

								<h2 class="major">All Publications</h2>
									<ul>
										
										<li><a href="https://doi.org/10.1016/j.isci.2025.113448"><b>NovumRNA: Accurate prediction of non-canonical tumor antigens from RNA sequencing data</b></a><br/>Ausserhofer M, Rieder D, Facciolla M, Gronauer R, Lamberti G, Lisandrelli R, Pellegatta S, Trajanoski Z, Finotello F<br/><b>iScience</b>, in print (2025)</li><br/>
										<li><a href="https://doi.org/10.1016/bs.mcb.2025.01.003"><b>Next-generation deconvolution of the tumor microenvironment with omnideconv</b></a><br/>Merotto L, Dietrich A, List M, Finotello F<br/><b>Methods Cell Biol</b>, 196:87-112 (2025)</li><br/>
										<li><a href="https://doi.org/10.1016/bs.mcb.2025.01.007"><b>Comprehensive prediction of tumor neoantigens with nextNEOpi</b></a><br/>Ausserhofer M, Rieder D, Finotello F<br/><b>Methods Cell Biol</b>, 196:113-137 (2025)</li><br/>
										<li><a href="https://doi.org/10.1038/s43018-025-01007-z"><b>IL-17A-secreting γδ T cells promote resistance to CDK4/CDK6 inhibitors in HR(+)HER2(-) breast cancer via CX3CR1(+) macrophages</b></a><br/>Petroni G, Galassi C, Gouin KH 3rd, Chen HH, Buqué A, Bloy N, Yamazaki T, Sato A, Beltrán-Visiedo M, Campia G, Jiménez-Cortegana C, Shah A, Kirchmair A, Massa C, Wickenhauser C, de Andrea CE, Navarro-Rubio B, Serrano-Mendioroz I, Navarro Manzano E, Satty AM, Rippon B, Finotello F, Trajanoski Z, Zhou XK, Scandura JM, García-Martínez E, Ayala de la Peña F, Rodríguez-Ruiz ME, Seliger B, Sánchez-Margalet V, de la Cruz-Merino L, Basho RK, Shiao SL, McArthur HL, Formenti SC, Knott SRV, Galluzzi L<br/><b>Nat Cancer</b>. 2025 Jul 7 (2025)</li><br/>
										<li><a href="https://doi.org/10.1016/j.crmeth.2025.100985"><b>Mathematically mapping the network of cells in the tumor microenvironment</b></a><br/>van Santvoort M, Lapuente-Santana Ó, Zopoglou M, Zackl C, Finotello F, van der Hoorn P, Eduati F<br/><b>Cell Rep Methods</b>, 5(2):100985 (2025)</li><br/>
										<li><a href="https://doi.org/10.1016/j.isci.2024.110529"><b>Multimodal analysis unveils tumor microenvironment heterogeneity linked to immune activity and evasion</b></a><br/>Lapuente-Santana Ó, Sturm G, Kant J, Ausserhofer M, Zackl C, Zopoglou M, McGranahan N, Rieder D, Trajanoski Z, da Cunha Carvalho de Miranda NF, Eduati F, Finotello F<br/><b>iScience</b>, 27(8):110529 (2024)</li><br/>
										<li><a href="https://doi.org/10.1038/s41591-024-02978-9"><b>A spatial architecture-embedding HLA signature to predict clinical response to immunotherapy in renal cell carcinoma</b></a><br/>Kinget L, Naulaerts S, Govaerts J, Vanmeerbeek I, Sprooten J, Laureano RS, Dubroja N, Shankar G, Bosisio FM, Roussel E, Verbiest A, Finotello F, Ausserhofer M, Lambrechts D, Boeckx B, Wozniak A, Boon L, Kerkhofs J, Zucman-Rossi J, Albersen M, Baldewijns M, Beuselinck B, Garg AD<br/><b>Nat Med</b>, 30(6):1667-1679 (2024)</li><br/>
										<li><a href="https://doi.org/10.1136/jitc-2023-008606"><b>Tumor-targeted therapy with BRAF-inhibitor recruits activated dendritic cells to promote tumor immunity in melanoma</b></a><br/>Hornsteiner F, Vierthaler J, Strandt H, Resag A, Fu Z, Ausserhofer M, Tripp CH, Dieckmann S, Kanduth M, Farrand K, Bregar S, Nemati N, Hermann-Kleiter N, Seretis A, Morla S, Mullins D, Finotello F, Trajanoski Z, Wollmann G, Ronchese F, Schmitz M, Hermans IF, Stoitzner P<br/><b>J Immunother Cancer</b>, 12(4):e008606 (2024)</li><br/>
										<li><a href="https://doi.org/10.1007/s12015-024-10698-3"><b>Single-cell Profiling of Reprogrammed Human Neural Stem Cells Unveils High Similarity to Neural Progenitors in the Developing Central Nervous System</b></a><br/>Spathopoulou A, Podlesnic M, De Gaetano L, Kirsch EM, Tisch M, Finotello F, Aigner L, Günther K, Edenhofer F<br/><b>Stem Cell Rev Rep</b>, 20(5):1325-1339 (2024)</li><br/>
										<li><a href="https://doi.org/10.1093/bioadv/vbae032"><b>Making mouse transcriptomics deconvolution accessible with immunedeconv</b></a><br/>Merotto L, Sturm G, Dietrich A, List M, Finotello F<br/><b>Bioinform Adv</b>, 4(1):vbae032 (2024)</li><br/>
										<li><a href="https://doi.org/10.1016/j.tranon.2023.101857"><b>The highly selective and oral phosphoinositide 3-kinase delta (PI3K-δ) inhibitor roginolisib induces apoptosis in mesothelioma cells and increases immune effector cell composition</b></a><br/>Kalla C, Ott G, Finotello F, Niewola-Staszkowska K, Conza GD, Lahn M, van der Veen L, Schüler J, Falkenstern-Ge R, Kopecka J, Riganti C<br/><b>Transl Oncol</b>, 43:101857 (2024)</li><br/>
										<li><a href="https://doi.org/10.1186/s12967-023-04705-3"><b>A community challenge to predict clinical outcomes after immune checkpoint blockade in non-small cell lung cancer</b></a><br/>Mason M, Lapuente-Santana Ó, Halkola AS, Wang W, Mall R, Xiao X, Kaufman J, Fu J, Pfeil J, Banerjee J, Chung V, Chang H, Chasalow SD, Lin HY, Chai R, Yu T, Finotello F, Mirtti T, Mäyränpää MI, Bao J, Verschuren EW, Ahmed EI, Ceccarelli M, Miller LD, Monaco G, Hendrickx WRL, Sherif S, Yang L, Tang M, Gu SS, Zhang W, Zhang Y, Zeng Z, Das Sahu A, Liu Y, Yang W, Bedognetti D, Tang J, Eduati F, Laajala TD, Geese WJ, Guinney J, Szustakowski JD, Vincent BG, Carbone DP<br/><b>J Transl Med</b>, 22(1):190 (2024)</li><br/>
										<li><a href="https://doi.org/10.1016/bs.ircmb.2023.05.002"><b>Next-generation deconvolution of transcriptomic data to investigate the tumor microenvironment</b></a><br/>Merotto L, Zopoglou M, Zackl C, Finotello F<br/><b>Int Rev Cell Mol Biol</b>, 382:103-143 (2024)</li><br/>
										<li><a href="https://doi.org/10.1016/j.isci.2023.108399"><b>Functional and spatial proteomics profiling reveals intra- and intercellular signaling crosstalk in colorectal cancer</b></a><br/>Plattner C, Lamberti G, Blattmann P, Kirchmair A, Rieder D, Loncova Z, Sturm G, Scheidl S, Ijsselsteijn M, Fotakis G, Noureen A, Lisandrelli R, Böck N, Nemati N, Krogsdam A, Daum S, Finotello F, Somarakis A, Schäfer A, Wilflingseder D, Gonzalez Acera M, Öfner D, Huber LA, Clevers H, Becker C, Farin HF, Greten FR, Aebersold R, de Miranda NFCC, Trajanoski Z<br/><b>iScience</b>, 26(12):108399 (2023)</li><br/>
										<li><a href="https://doi.org/10.1038/s41421-023-00605-4"><b>Single cell dynamics of tumor specificity vs bystander activity in CD8(+) T cells define the diverse immune landscapes in colorectal cancer</b></a><br/>Borràs DM, Verbandt S, Ausserhofer M, Sturm G, Lim J, Verge GA, Vanmeerbeek I, Laureano RS, Govaerts J, Sprooten J, Hong Y, Wall R, De Hertogh G, Sagaert X, Bislenghi G, D'Hoore A, Wolthuis A, Finotello F, Park WY, Naulaerts S, Tejpar S, Garg AD<br/><b>Cell Discov</b>, 9(1):114 (2023)</li><br/>
										<li><a href="https://doi.org/10.1038/s41587-023-01733-8"><b>The scverse project provides a computational ecosystem for single-cell omics data analysis</b></a><br/>Virshup I, Bredikhin D, Heumos L, Palla G, Sturm G, Gayoso A, Kats I, Koutrouli M; Scverse Community; Berger B, Pe'er D, Regev A, Teichmann SA, Finotello F, Wolf FA, Yosef N, Stegle O, Theis FJ<br/><b>Nat Biotechnol</b>, 41(5):604-606 (2023)</li><br/>
										<li><a href="https://doi.org/10.1128/spectrum.02944-22"><b>Role of the Extracytoplasmic Function Sigma Factor SigE in the Stringent Response of Mycobacterium tuberculosis</b></a><br/>Baruzzo G, Serafini A, Finotello F, Sanavia T, Cioetto-Mazzabò L, Boldrin F, Lavezzo E, Barzon L, Toppo S, Provvedi R, Manganelli R, Di Camillo B<br/><b>Microbiol Spectr</b>, 11(2):e0294422 (2023)</li><br/>
										<li><a href="https://doi.org/10.1093/bioadv/vbac092"><b>MAST: a hybrid Multi-Agent Spatio-Temporal model of tumor microenvironment informed using a data-driven approach</b></a><br/>Cesaro G, Milia M, Baruzzo G, Finco G, Morandini F, Lazzarini A, Alotto P, da Cunha Carvalho de Miranda NF, Trajanoski Z, Finotello F, Di Camillo B<br/><b>Bioinform Adv</b>, 2(1):vbac092 (2022)</li><br/>
										<li><a href="https://doi.org/10.1016/j.ccell.2022.10.008"><b>High-resolution single-cell atlas reveals diversity and plasticity of tissue-resident neutrophils in non-small cell lung cancer</b></a><br/>Salcher S, Sturm G, Horvath L, Untergasser G, Kuempers C, Fotakis G, Panizzolo E, Martowicz A, Trebo M, Pall G, Gamerith G, Sykora M, Augustin F, Schmitz K, Finotello F, Rieder D, Perner S, Sopper S, Wolf D, Pircher A, Trajanoski Z<br/><b>Cancer Cell</b>, 40(12):1503-1520.e8 (2022)</li><br/>
										<li><a href="https://doi.org/10.1016/j.cels.2022.08.006"><b>What are the current driving questions in immune repertoire research?</b></a><br/>Greiff V, Luning Prak ET, O'Donnell T, Finotello F, Reddy ST, Walczak A, Mora T<br/><b>Cell Syst</b>, 13(9):683-686 (2022)</li><br/>
										<li><a href="https://doi.org/10.1093/bioinformatics/btac499"><b>SimBu: bias-aware simulation of bulk RNA-seq data with variable cell-type composition</b></a><br/>Dietrich A, Sturm G, Merotto L, Marini F, Finotello F, List M<br/><b>Bioinformatics</b>, 38(Suppl_2):ii141-ii147 (2022)</li><br/>
										<li><a href="https://doi.org/10.1136/jitc-2021-004346"><b>Tumor-specific T cells support chemokine-driven spatial organization of intratumoral immune microaggregates needed for long survival</b></a><br/>Abdulrahman Z, Santegoets SJ, Sturm G, Charoentong P, Ijsselsteijn ME, Somarakis A, Höllt T, Finotello F, Trajanoski Z, van Egmond SL, Mustafa DAM, Welters MJP, de Miranda NFCC, van der Burg SH<br/><b>J Immunother Cancer</b>, 10(2):e004346 (2022)</li><br/>
										<li><a href="https://doi.org/10.1136/jitc-2021-003995"><b>CD161 expression and regulation defines rapidly responding effector CD4+ T cells associated with improved survival in HPV16-associated tumors</b></a><br/>Duurland CL, Santegoets SJ, Abdulrahman Z, Loof NM, Sturm G, Wesselink TH, Arens R, Boekestijn S, Ehsan I, van Poelgeest MIE, Finotello F, Hackl H, Trajanoski Z, Ten Dijke P, Braud VM, Welters MJP, van der Burg SH<br/><b>J Immunother Cancer</b>, 10(1):e003995 (2022)</li><br/>
										<li><a href="https://doi.org/10.1093/bioinformatics/btab759"><b>nextNEOpi: a comprehensive pipeline for computational neoantigen prediction</b></a><br/>Rieder D, Fotakis G, Ausserhofer M, René G, Paster W, Trajanoski Z, Finotello F<br/><b>Bioinformatics</b>, 38(4):1131-1132 (2022)</li><br/>
										<li><a href="https://doi.org/10.1002/ijc.33859"><b>NKG2A is a late immune checkpoint on CD8 T cells and marks repeated stimulation and cell division</b></a><br/>Borst L, Sluijter M, Sturm G, Charoentong P, Santegoets SJ, van Gulijk M, van Elsas MJ, Groeneveldt C, van Montfoort N, Finotello F, Trajanoski Z, Kiełbasa SM, van der Burg SH, van Hall T<br/><b>Int J Cancer</b>, 150(4):688-704 (2022)</li><br/>
										<li><a href="https://doi.org/10.1038/s41418-021-00853-5"><b>Stress-induced inflammation evoked by immunogenic cell death is blunted by the IRE1α kinase inhibitor KIRA6 through HSP60 targeting</b></a><br/>Rufo N, Korovesis D, Van Eygen S, Derua R, Garg AD, Finotello F, Vara-Perez M, Rožanc J, Dewaele M, de Witte PA, Alexopoulos LG, Janssens S, Sinkkonen L, Sauter T, Verhelst SHL, Agostinis P<br/><b>Cell Death Differ</b>, 29(1):230-245 (2022)</li><br/>
										<li><a href="https://doi.org/10.1016/j.patter.2021.100293"><b>Interpretable systems biomarkers predict response to immune-checkpoint inhibitors</b></a><br/>Lapuente-Santana Ó, van Genderen M, Hilbers PAJ, Finotello F, Eduati F<br/><b>Patterns (N Y)</b>Patterns, 2(8):100293 (2021)</li><br/>
										<li><a href="https://doi.org/10.1186/s13046-021-02023-4"><b>A vision of immuno-oncology: the Siena think tank of the Italian network for tumor biotherapy (NIBIT) foundation</b></a><br/>Maio M, Lahn M, Di Giacomo AM, Covre A, Calabrò L, Ibrahim R, Fox B; Siena Think Tank<br/><b>J Exp Clin Cancer Res</b>, 40(1):240 (2021)</li><br/>
										<li><a href="https://doi.org/10.3389/fonc.2020.01768"><b>Editorial: Multi-omic Data Integration in Oncology</b></a><br/>Finotello F, Calura E, Risso D, Hautaniemi S, Romualdi C<br/><b>Front Oncol</b>, 10:1768 (2020)</li><br/>
										<li><a href="https://doi.org/10.1158/2326-6066.CIR-20-0270"><b>CD39 Identifies the CD4(+) Tumor-Specific T-cell Population in Human Cancer</b></a><br/>Kortekaas KE, Santegoets SJ, Sturm G, Ehsan I, van Egmond SL, Finotello F, Trajanoski Z, Welters MJP, van Poelgeest MIE, van der Burg SH<br/><b>Cancer Immunol Res</b>, 8(10):1311-1321 (2020)</li><br/>
										<li><a href="https://doi.org/10.1038/s41590-020-0751-0"><b>Mitochondrial DNA drives abscopal responses to radiation that are inhibited by autophagy</b></a><br/>Yamazaki T, Kirchmair A, Sato A, Buqué A, Rybstein M, Petroni G, Bloy N, Finotello F, Stafford L, Navarro Manzano E, Ayala de la Peña F, García-Martínez E, Formenti SC, Trajanoski Z, Galluzzi L<br/><b>Nat Immunol</b>, 21(10):1160-1171 (2020)</li><br/>
										<li><a href="https://doi.org/10.1093/bioinformatics/btaa611"><b>Scirpy: a Scanpy extension for analyzing single-cell T-cell receptor-sequencing data</b></a><br/>Sturm G, Szabo T, Fotakis G, Haider M, Rieder D, Trajanoski Z, Finotello F<br/><b>Bioinformatics</b>, 36(18):4817-4818 (2020)</li><br/>
										<li><a href="https://doi.org/10.1016/bs.mie.2019.05.056"><b>Deconvoluting tumor-infiltrating immune cells from RNA-seq data using quanTIseq</b></a><br/>Plattner C, Finotello F, Rieder D<br/><b>Methods Enzymol</b>, 636:261-285 (2020)</li><br/>
										<li><a href="https://doi.org/10.1007/978-1-0716-0327-7_16"><b>Immunedeconv: An R Package for Unified Access to Computational Methods for Estimating Immune Cell Fractions from Bulk RNA-Sequencing Data</b></a><br/>Sturm G, Finotello F, List M<br/><b>Methods Mol Biol</b>, 2120:223-232 (2020)</li><br/>
										<li><a href="https://doi.org/10.1007/978-1-0716-0327-7_15"><b>In Silico Cell-Type Deconvolution Methods in Cancer Immunotherapy</b></a><br/>Sturm G, Finotello F, List M<br/><b>Methods Mol Biol</b>, 2120:213-222 (2020)</li><br/>
										<li><a href="https://doi.org/10.1007/978-1-0716-0327-7_9"><b>In Silico Prediction of Tumor Neoantigens with TIminer</b></a><br/>Kirchmair A, Finotello F<br/><b>Methods Mol Biol</b>, 2120:129-145 (2020)</li><br/>
										<li><a href="https://doi.org/10.1093/bioinformatics/btz879"><b>NeoFuse: predicting fusion neoantigens from RNA sequencing data</b></a><br/>Fotakis G, Rieder D, Haider M, Trajanoski Z, Finotello F<br/><b>Bioinformatics</b>, 36(7):2260-2261 (2020)</li><br/>
										<li><a href="https://doi.org/10.1158/1078-0432.CCR-19-1335"><b>Guadecitabine Plus Ipilimumab in Unresectable Melanoma: The NIBIT-M4 Clinical Trial</b></a><br/>Di Giacomo AM, Covre A, Finotello F, Rieder D, Danielli R, Sigalotti L, Giannarelli D, Petitprez F, Lacroix L, Valente M, Cutaia O, Fazio C, Amato G, Lazzeri A, Monterisi S, Miracco C, Coral S, Anichini A, Bock C, Nemc A, Oganesian A, Lowder J, Azab M, Fridman WH, Sautès-Fridman C, Trajanoski Z, Maio M<br/><b>Clin Cancer Res</b>, 25(24):7351-7362 (2019)</li><br/>
										<li><a href="https://doi.org/10.1038/s41576-019-0166-7"><b>Next-generation computational tools for interrogating cancer immunity</b></a><br/>Finotello F, Rieder D, Hackl H, Trajanoski Z<br/><b>Nat Rev Genet</b>, 20(12):724-746 (2019)</li><br/>
										<li><a href="https://doi.org/10.1093/bioinformatics/btz363"><b>Comprehensive evaluation of transcriptome-based cell-type quantification methods for immuno-oncology</b></a><br/>Sturm G, Finotello F, Petitprez F, Zhang JD, Baumbach J, Fridman WH, List M, Aneichyk T<br/><b>Bioinformatics</b>, 35(14):i436-i445 (2019)</li><br/>
										<li><a href="https://doi.org/10.1186/s13073-019-0638-6"><b>Molecular and pharmacological modulators of the tumor immune contexture revealed by deconvolution of RNA-seq data</b></a><br/>Finotello F, Mayer C, Plattner C, Laschober G, Rieder D, Hackl H, Krogsdam A, Loncova Z, Posch W, Wilflingseder D, Sopper S, Ijsselsteijn M, Brouwer TP, Johnson D, Xu Y, Wang Y, Sanders ME, Estrada MV, Ericsson-Gonzalez P, Charoentong P, Balko J, de Miranda NFDCC, Trajanoski Z<br/><b>Genome Med</b>, 11(1):34 (2019)</li><br/>
										<li><a href="https://doi.org/10.3389/fonc.2018.00430"><b>Multi-Omics Profiling of the Tumor Microenvironment: Paving the Way to Precision Immuno-Oncology</b></a><br/>Finotello F, Eduati F<br/><b>Front Oncol</b>, 8:430 (2018)</li><br/>
										<li><a href="https://doi.org/10.1186/s12859-018-2360-6"><b>Optimizing PCR primers targeting the bacterial 16S ribosomal RNA gene</b></a><br/>Sambo F, Finotello F, Lavezzo E, Baruzzo G, Masi G, Peta E, Falda M, Toppo S, Barzon L, Di Camillo B<br/><b>BMC Bioinformatics</b>, 19(1):343 (2018)</li><br/>
										<li><a href="https://doi.org/10.2337/db17-0914"><b>A Genome-Wide Association Study of Diabetic Kidney Disease in Subjects With Type 2 Diabetes</b></a><br/>van Zuydam NR, Ahlqvist E, Sandholm N, Deshmukh H, Rayner NW, Abdalla M, Ladenvall C, Ziemek D, Fauman E, Robertson NR, McKeigue PM, Valo E, Forsblom C, Harjutsalo V; Finnish Diabetic Nephropathy Study (FinnDiane); Perna A, Rurali E, Marcovecchio ML, Igo RP Jr, Salem RM, Perico N, Lajer M, Käräjämäki A, Imamura M, Kubo M, Takahashi A, Sim X, Liu J, van Dam RM, Jiang G, Tam CHT, Luk AOY, Lee HM, Lim CKP, Szeto CC, So WY, Chan JCN; Hong Kong Diabetes Registry Theme-based Research Scheme Project Group; Ang SF, Dorajoo R, Wang L, Clara TSH, McKnight AJ, Duffy S; Warren 3 and Genetics of Kidneys in Diabetes (GoKinD) Study Group; Pezzolesi MG; GENIE (GEnetics of Nephropathy an International Effort) Consortium; Marre M, Gyorgy B, Hadjadj S, Hiraki LT; Diabetes Control and Complications Trial (DCCT)/Epidemiology of Diabetes Interventions and Complications (EDIC) Research Group; Ahluwalia TS, Almgren P, Schulz CA, Orho-Melander M, Linneberg A, Christensen C, Witte DR, Grarup N, Brandslund I, Melander O, Paterson AD, Tregouet D, Maxwell AP, Lim SC, Ma RCW, Tai ES, Maeda S, Lyssenko V, Tuomi T, Krolewski AS, Rich SS, Hirschhorn JN, Florez JC, Dunger D, Pedersen O, Hansen T, Rossing P, Remuzzi G; SUrrogate markers for Micro- and Macrovascular hard endpoints for Innovative diabetes Tools (SUMMIT) Consortium; Brosnan MJ, Palmer CNA, Groop PH, Colhoun HM, Groop LC, McCarthy MI<br/><b>Diabetes</b>, 67(7):1414-1427 (2018)</li><br/>
										<li><a href="https://doi.org/10.1038/s41598-018-24437-5"><b>Deviations of the immune cell landscape between healthy liver and hepatocellular carcinoma</b></a><br/>Rohr-Udilova N, Klinglmüller F, Schulte-Hermann R, Stift J, Herac M, Salzmann M, Finotello F, Timelthaler G, Oberhuber G, Pinter M, Reiberger T, Jensen-Jarolim E, Eferl R, Trauner M<br/><b>Sci Rep</b>, 8(1):6220 (2018)</li><br/>
										<li><a href="https://doi.org/10.1007/s00262-018-2150-z"><b>Quantifying tumor-infiltrating immune cells from transcriptomics data</b></a><br/>Finotello F, Trajanoski Z<br/><b>Cancer Immunol Immunother</b>, 67(7):1031-1040 (2018)</li><br/>
										<li><a href="https://doi.org/10.1038/s41467-017-02424-0"><b>Targeting immune checkpoints potentiates immunoediting and changes the dynamics of tumor evolution</b></a><br/>Efremova M, Rieder D, Klepsch V, Charoentong P, Finotello F, Hackl H, Hermann-Kleiter N, Löwer M, Baier G, Krogsdam A, Trajanoski Z<br/><b>Nat Commun</b>, 9(1):32 (2018)</li><br/>
										<li><a href="https://doi.org/10.3389/fimmu.2017.01679"><b>Neoantigens Generated by Individual Mutations and Their Role in Cancer Immunity and Immunotherapy</b></a><br/>Efremova M, Finotello F, Rieder D, Trajanoski Z<br/><b>Front Immunol</b>, 8:1679 (2017)</li><br/>
										<li><a href="https://doi.org/10.1093/bioinformatics/btx377"><b>TIminer: NGS data mining pipeline for cancer immunology and immunotherapy</b></a><br/>Tappeiner E, Finotello F, Charoentong P, Mayer C, Rieder D, Trajanoski Z<br/><b>Bioinformatics</b>, 33(19):3140-3141 (2017)</li><br/>
										<li><a href="https://doi.org/10.1007/978-1-4939-6807-7_10"><b>Analysis of High-Throughput RNA Bisulfite Sequencing Data</b></a><br/>Rieder D, Finotello F<br/><b>Methods Mol Biol</b>, 1562:143-154 (2017)</li><br/>
										<li><a href="https://doi.org/10.1186/s13073-017-0402-8"><b>New strategies for cancer immunotherapy: targeting regulatory T cells</b></a><br/>Finotello F, Trajanoski Z<br/><b>Genome Med</b>, 9(1):10 (2017)</li><br/>
										<li><a href="https://doi.org/10.1016/j.celrep.2016.12.019"><b>Pan-cancer Immunogenomic Analyses Reveal Genotype-Immunophenotype Relationships and Predictors of Response to Checkpoint Blockade</b></a><br/>Charoentong P, Finotello F, Angelova M, Mayer C, Efremova M, Rieder D, Hackl H, Trajanoski Z<br/><b>Cell Rep</b>, 18(1):248-262 (2017)</li><br/>
										<li><a href="https://doi.org/10.1093/bib/bbw119"><b>Measuring the diversity of the human microbiota with targeted next-generation sequencing</b></a><br/>Finotello F, Mastrorilli E, Di Camillo B<br/><b>Brief Bioinform</b>, 19(4):679-692 (2018)</li><br/>
										<li><a href="https://doi.org/10.1038/nrg.2016.67"><b>Computational genomics tools for dissecting tumour-immune cell interactions</b></a><br/>Hackl H, Charoentong P, Finotello F, Trajanoski Z<br/><b>Nat Rev Genet</b>, 17(8):441-58 (2016)</li><br/>
										<li><a href="https://doi.org/10.1038/nmeth.3773"><b>Inferring causal molecular networks: empirical assessment through a community-based effort</b></a><br/>Hill SM, Heiser LM, Cokelaer T, Unger M, Nesser NK, Carlin DE, Zhang Y, Sokolov A, Paull EO, Wong CK, Graim K, Bivol A, Wang H, Zhu F, Afsari B, Danilova LV, Favorov AV, Lee WS, Taylor D, Hu CW, Long BL, Noren DP, Bisberg AJ; HPN-DREAM Consortium; Mills GB, Gray JW, Kellen M, Norman T, Friend S, Qutub AA, Fertig EJ, Guan Y, Song M, Stuart JM, Spellman PT, Koeppl H, Stolovitzky G, Saez-Rodriguez J, Mukherjee S<br/><b>Nat Methods</b>, 13(4):310-8 (2016)</li><br/>
										<li><a href="https://doi.org/10.1109/EMBC.2015.7319309"><b>EEG signal features extraction based on fractal dimension</b></a><br/>Finotello F, Scarpa F, Zanon M<br/><b>Annu Int Conf IEEE Eng Med Biol Soc</b>, 2015:4154-7 (2015)</li><br/>
										<li><a href="https://doi.org/10.1186/1471-2164-16-S6-S2"><b>FunPat: function-based pattern analysis on RNA-seq time series data</b></a><br/>Sanavia T, Finotello F, Di Camillo B<br/><b>BMC Genomics</b>, 16(Suppl 6):S2 (2015)</li><br/>
										<li><a href="https://doi.org/10.1093/bfgp/elu035"><b>Measuring differential gene expression with RNA-seq: challenges and strategies for data analysis</b></a><br/>Finotello F, Di Camillo B<br/><b>Brief Funct Genomics</b>, 14(2):130-42 (2015)</li><br/>
										<li><a href="https://doi.org/10.1073/pnas.1402665111"><b>Global genomic and transcriptomic analysis of human pancreatic islets reveals novel genes influencing glucose metabolism</b></a><br/>Fadista J, Vikman P, Laakso EO, Mollet IG, Esguerra JL, Taneera J, Storm P, Osmark P, Ladenvall C, Prasad RB, Hansson KB, Finotello F, Uvebrant K, Ofori JK, Di Camillo B, Krus U, Cilio CM, Hansson O, Eliasson L, Rosengren AH, Renström E, Wollheim CB, Groop L<br/><b>Proc Natl Acad Sci U S A</b>, 111(38):13924-9 (2014)</li><br/>
										<li><a href="https://doi.org/10.1186/1471-2105-15-S1-S7"><b>Reducing bias in RNA sequencing data: a novel approach to compute counts</b></a><br/>Finotello F, Lavezzo E, Bianco L, Barzon L, Mazzon P, Fontana P, Toppo S, Di Camillo B<br/><b>BMC Bioinformatics</b>, 15 Suppl 1(Suppl 1):S7 (2014)</li><br/>
										<li><a href="https://doi.org/10.1186/1471-2334-13-554"><b>Genomic comparative analysis and gene function prediction in infectious diseases: application to the investigation of a meningitis outbreak</b></a><br/>Lavezzo E, Toppo S, Franchin E, Di Camillo B, Finotello F, Falda M, Manganelli R, Palù G, Barzon L<br/><b>BMC Infect Dis</b>, 13:554 (2013)</li><br/>
										<li><a href="https://doi.org/10.1093/bib/bbr063"><b>Comparative analysis of algorithms for whole-genome assembly of pyrosequencing data</b></a><br/>Finotello F, Lavezzo E, Fontana P, Peruzzo D, Albiero A, Barzon L, Falda M, Di Camillo B, Toppo S<br/><b>Brief Bioinform</b>, 13(3):269-80 (2012)</li><br/>
										<li><a href="https://doi.org/10.1128/JB.00789-10"><b>Draft genome sequences of two Neisseria meningitidis serogroup C clinical isolates</b></a><br/>Lavezzo E, Toppo S, Barzon L, Cobelli C, Di Camillo B, Finotello F, Franchin E, Peruzzo D, Toffolo GM, Trevisan M, Palù G<br/><b>J Bacteriol</b>, 192(19):5270-1 (2010)</li><br/>

								</ul>
							</div>
						</section>

				<!-- Footer -->
					<section id="footer">
						<div class="inner">
							<ul class="copyright">
								<li>Francesca Finotello. All rights reserved.</li>
								<li>Design: <a href="http://html5up.net">HTML5 UP</a>.</li>
								<li>Photos: <a href="https://www.colourbox.de/">Colourbox.de</a> (Kiyoshi Takahase Segundo); 
									<a href="https://unsplash.com/">Unsplash</a>; David Bullock.
								</li>	
							</ul>
						</div>
					</section>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>